src/gui/EditWidgetIcons.cpp

```diff
@@ -66,6 +66,15 @@ EditWidgetIcons::EditWidgetIcons(QWidget* parent)
     connect(m_ui->deleteButton, SIGNAL(clicked()), SLOT(removeCustomIcon()));
     connect(m_ui->faviconButton, SIGNAL(clicked()), SLOT(downloadFavicon()));
 
+    connect(m_ui->defaultIconsRadio, SIGNAL(toggled(bool)), this, SIGNAL(widgetUpdated()));
+    connect(m_ui->defaultIconsRadio, SIGNAL(toggled(bool)), this, SIGNAL(widgetUpdated()));
+    connect(m_ui->defaultIconsView->selectionModel(),
+            SIGNAL(selectionChanged(const QItemSelection &, const QItemSelection &)),
```

Comment:
No space before (please change that everywhere)

Reply:
You mean the newline? So should it be like:
```
    connect(m_ui->defaultIconsView->selectionModel(), SIGNAL(selectionChanged(const QItemSelection &, const QItemSelection &)),
            this, SIGNAL(widgetUpdated()));
```
or like this:
```
    connect(m_ui->defaultIconsView->selectionModel(), SIGNAL(selectionChanged(const QItemSelection &, const QItemSelection &)), this, SIGNAL(widgetUpdated()));
```

Reply:
I meant no space before &amp;. Github swallowed my ubeacaoed ampersand.